### PR TITLE
Clarify the fields that is available only for Bot Mgmt Ent customer (Botxxx, JA3)

### PR DESCRIPTION
**Pull request by kyouheicf**

### CODE REVIEW

**Comment by angelampcosta** [comment]
Hi @kyouheicf The log field files are Code generated by logshare-api. They cannot be edited directly in Github. Just to confirm, did you do that update first?

**Comment by kyouheicf** [comment]
@pedrosousa Can you help move this forward as we did the previous one?
https://github.com/cloudflare/cloudflare-docs/pull/8504

**Review by soheiokamoto — APPROVED** [review]

**Review by soheiokamoto — CHANGES_REQUESTED** [review]
The descriptions are slightly different than the changes made at the source repo.
Please update this PR to match them.

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ The descriptions below detail the fields available for `http_requests`.
 
 | Field | Value | Type |
 | -- | -- | -- |
-| BotDetectionIDs | List of IDs that correlate to the Bot Management Heuristic detections made on a request. Available in Logpush v2 only. | array[int] |
-| BotScore | Cloudflare Bot Score. Scores below 30 are commonly associated with automated traffic. Available for Bot Management customers (please contact your account team to enable). | int |
-| BotScoreSrc | Detection engine responsible for generating the Bot Score. <br />Possible values are <em>Not Computed</em> \| <em>Heuristics</em> \| <em>Machine Learning</em> \| <em>Behavioral Analysis</em> \| <em>Verified Bot</em> \| <em>JS Fingerprinting</em> \| <em>Cloudflare Service</em>. | string |
-| BotTags | Type of bot traffic (if available). Refer to [Bot Tags](/bots/concepts/cloudflare-bot-tags/) for the list of potential values. Available in Logpush v2 only. | array[string] |
+| BotDetectionIDs | List of IDs that correlate to the Bot Management Heuristic detections made on a request. Available in Logpush v2 only. Available only for Bot Management customers (please contact your account team to enable). | array[int] |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/zone/http_requests.md]:
```suggestion
| BotDetectionIDs | List of IDs that correlate to the Bot Management Heuristic detections made on a request. Available in Logpush v2 only. Available only for Bot Management customers. To enable this feature, contact your account team. | array[int] |
```

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ The descriptions below detail the fields available for `http_requests`.
...
+| BotScore | Cloudflare Bot Score. Scores below 30 are commonly associated with automated traffic. Available only for Bot Management customers (please contact your account team to enable). | int |
+| BotScoreSrc | Detection engine responsible for generating the Bot Score. <br />Possible values are <em>Not Computed</em> \| <em>Heuristics</em> \| <em>Machine Learning</em> \| <em>Behavioral Analysis</em> \| <em>Verified Bot</em> \| <em>JS Fingerprinting</em> \| <em>Cloudflare Service</em>. Available only for Bot Management customers (please contact your account team to enable). | string |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/zone/http_requests.md]:
```suggestion
| BotScoreSrc | Detection engine responsible for generating the Bot Score. <br />Possible values are <em>Not Computed</em> \| <em>Heuristics</em> \| <em>Machine Learning</em> \| <em>Behavioral Analysis</em> \| <em>Verified Bot</em> \| <em>JS Fingerprinting</em> \| <em>Cloudflare Service</em>. Available only for Bot Management customers. To enable this feature, contact your account team. | string |
```

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 3]
```diff
@@ -14,10 +14,10 @@ The descriptions below detail the fields available for `http_requests`.
...
+| BotTags | Type of bot traffic (if available). Refer to [Bot Tags](/bots/concepts/cloudflare-bot-tags/) for the list of potential values. Available in Logpush v2 only. Available only for Bot Management customers (please contact your account team to enable). | array[string] |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/zone/http_requests.md]:
```suggestion
| BotTags | Type of bot traffic (if available). Refer to [Bot Tags](/bots/concepts/cloudflare-bot-tags/) for the list of potential values. Available in Logpush v2 only. Available only for Bot Management customers. To enable this feature, contact your account team. | array[string] |
```

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 4]
```diff
@@ -14,10 +14,10 @@ The descriptions below detail the fields available for `http_requests`.
...
-| BotScore | Cloudflare Bot Score. Scores below 30 are commonly associated with automated traffic. Available for Bot Management customers (please contact your account team to enable). | int |
-| BotScoreSrc | Detection engine responsible for generating the Bot Score. <br />Possible values are <em>Not Computed</em> \| <em>Heuristics</em> \| <em>Machine Learning</em> \| <em>Behavioral Analysis</em> \| <em>Verified Bot</em> \| <em>JS Fingerprinting</em> \| <em>Cloudflare Service</em>. | string |
-| BotTags | Type of bot traffic (if available). Refer to [Bot Tags](/bots/concepts/cloudflare-bot-tags/) for the list of potential values. Available in Logpush v2 only. | array[string] |
+| BotDetectionIDs | List of IDs that correlate to the Bot Management Heuristic detections made on a request. Available in Logpush v2 only. Available only for Bot Management customers (please contact your account team to enable). | array[int] |
+| BotScore | Cloudflare Bot Score. Scores below 30 are commonly associated with automated traffic. Available only for Bot Management customers (please contact your account team to enable). | int |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/zone/http_requests.md]:
```suggestion
| BotScore | Cloudflare Bot Score. Scores below 30 are commonly associated with automated traffic. Available only for Bot Management customers. To enable this feature, contact your account team. | int |
```

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ The descriptions below detail the fields available for `http_requests`.
...
-| JA3Hash | The MD5 hash of the JA3 fingerprint used to profile SSL/TLS clients. Available in Logpush v2 only. | string |
+| JA3Hash | The MD5 hash of the JA3 fingerprint used to profile SSL/TLS clients. Available in Logpush v2 only. Available only for Bot Management customers (please contact your account team to enable). | string |
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/zone/http_requests.md]:
```suggestion
| JA3Hash | The MD5 hash of the JA3 fingerprint used to profile SSL/TLS clients. Available in Logpush v2 only. Available only for Bot Management customers. To enable this feature, contact your account team. | string |
```

**Review by soheiokamoto — APPROVED** [review]

**Pull request merged**